Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
@greg0ire I thought it was Symfony that replaced the `.` per a `__` but it comes from sonata in fact https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Form/FormMapper.php#L77